wear/src/main/java/com/eveningoutpost/dexdrip/Services/CustomComplicationProviderService.java

```diff
@@ -46,6 +48,7 @@
     private static final String TAG = "ComplicationProvider";
     private static final long STALE_MS = Constants.MINUTE_IN_MS * 15;
     private static final long FRESH_MS = Constants.MINUTE_IN_MS * 5;
+    private String externalStatusString = "no status";
```

Comment:
Does this string immediately get excluded from scope by the declaration in line 137? Shouldn't this line be removed? Also I think you can optimize imports and remove the Datamap import.